collector.go

```diff
@@ -17,7 +17,7 @@ package main
 import (
 	"net"
 
-	"github.com/prometheus-community/pro-bing"
+	probing "github.com/prometheus-community/pro-bing"
```

Comment:
The package is already `probing`, not sure why you're changing the alias here.

Reply:
I'll remove it. I believe that was introduced automatically by visual studio code.

Reply:
Yup. VSCode kept modifying it on "save". Fixed.

Reply:
Looks like one got missed.